tasks/packages_install.yml

```diff
@@ -24,6 +24,19 @@
     - python3-venv
   when: python_major==3
 
+- name: install sqlite database on the host
+  apt: name="sqlite3" update_cache=yes state=present cache_valid_time=3600
+  when: database=="sqlite3"
+
+- block:
+  - name: install postgres database on the host
+    apt: name="postgresql-{{postgresql_version}}" update_cache=yes state=present cache_valid_time=3600
```

Comment:
`ansible-lint` will warn you about those lines. Also if you install `python3-psycopg2` (if `ansible_python_interpreter` is set to `/usr/bin/python3`). You can afterwards use something like this instead of running shell commands:

```
    - name: Creating the DB user
      become_user: postgres
      postgresql_user:
        name: "{{ db_user }}"
        password: "{{ db_user_pass }}"
        role_attr_flags: NOSUPERUSER

    - name: Creating the database
      become_user: postgres
      postgresql_db:
        name: "{{ db_name }}"
        state: present
        encoding: UTF-8
        owner: "{{ db_user }}"
```

Reply:
Nice! This might simplify the followup PR.

PS: I will move the further changes in a feature branch.